_posts/en/newsletters/2020-07-15-newsletter.md

```diff
@@ -0,0 +1,119 @@
+---
+title: 'Bitcoin Optech Newsletter #106'
+permalink: /en/newsletters/2020/07/15/
+name: 2020-07-15-newsletter
+slug: 2020-07-15-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposed update to the draft BIP118
+`SIGHASH_NOINPUT` and summarizes notable changes to popular Bitcoin
+infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **BIP118 update:** Anthony Towns [posted][towns post] to the
+  Bitcoin-Dev mailing list a link to a PR that proposes to replace the
+  existing text of the [BIP118][] draft of [SIGHASH_NOINPUT][topic
+  sighash_noinput] with the [draft specification][anyprevout spec] for
+  `SIGHASH_ANYPREVOUT` and `SIGHASH_ANYPREVOUTANYSCRIPT`.  Both
+  proposals describe optional signature hash (sighash) flags that do not
+  commit to the particular UTXOs (inputs/previous outputs) being spent
+  in a transaction, making it possible to create a signature for a
+  transaction without knowing what UTXO it will spend.  That feature can
+  be used by the proposed [eltoo][topic eltoo] settlement layer to allow
+  creating a series of transactions where any later transaction can
+  spend the value from any earlier transaction, allowing an offchain
+  contract to be settled in its latest state even if earlier states get
+  confirmed onchain.
+
+    The main difference between the noinput and anyprevout proposals is
+    that noinput would require its own new version of segwit but
+    anyprevout uses one of the upgrade features from the proposed [BIP342][]
+    specification of [tapscript][topic tapscript].  Additional
+    differences between the proposals are described in the [revised
+    text][anyprevout revisions] itself.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19219][] clarifies the distinction between manual peer banning
```

Comment:
Tried to find a good place in the current text to add the link @jnewbery mentioned and "distinction" might be a candidate, thought it breaks up the sentence a bit. No strong preference.